src/components/Contact/Contact.tsx

```diff
@@ -40,12 +41,14 @@ export const Contact = ({
   institutions,
   contactRole,
   name,
+  nested,
   teamTitle,
   teamUrl,
   tel
 }: ContactProps) => {
   const classNames = cx('cc-contact', {
-    [`${className}`]: className
+    'cc-contact--nested': nested,
```

Comment:
Does this selector have any style rules attached to it? I can't see any in the _contact.scss file... If we have no styles for it, I think we should remove it to avoid confusion.

Reply:
not yet, I am waiting for the designs. I wanted to include it as a placeholder for now.

Reply:
I create another PR with designs for nested contact us styles. I guess I can remove it from this PR.

Reply:
Yes, that makes sense - I'll approve this PR